src/js/extensions/paste.js

```diff
@@ -60,7 +60,10 @@
             [new RegExp(/<\/?o:[a-z]*>/gi), ''],
 
             // Microsoft Word adds some special elements around list items
-            [new RegExp(/<!\[if !supportLists\]>(((?!<!).)*)<!\[endif]\>/gi), '$1']
+            [new RegExp(/<!\[if !supportLists\]>(((?!<!).)*)<!\[endif]\>/gi), '$1'],
+
+            // remove font tag
+            [new RegExp(/<font[^><]*>|<.font[^><]*>/gi), '']
```

Comment:
@tomasfrancisco I actually though that cleaning the style off of every string was he purpose of `forcePlainText: true` option.  My interpretation of `cleanPastedHTML` had been to remove unwanted tags, or to try to convert unusual tags into more standard ones.